packages/paste-core/components/form-pill-group/src/FormPillGroup.tsx

```diff
@@ -66,14 +73,14 @@ const SizeStyles: Record<FormPillGroupSizeVariant, Pick<BoxProps, "columnGap" |
 };
 
 const FormPillGroupStyles = React.forwardRef<HTMLUListElement, FormPillGroupProps>(
-  ({ element = "FORM_PILL_GROUP", display = "flex", size = "default", ...props }, ref) => {
+  ({ element = "FORM_PILL_GROUP", display = "flex", size = "default", variant = "default", ...props }, ref) => {
```

Comment:
suggestion: call the default variant "listbox" instead of "default" (but still make it the default)?